sources/packages/backend/apps/queue-consumers/src/processors/schedulers/sfas-integration/sims-to-sfas-integration.scheduler.ts

```diff
@@ -0,0 +1,58 @@
+import { InjectQueue, Process, Processor } from "@nestjs/bull";
+import { QueueService } from "@sims/services/queue";
+import { Job, Queue } from "bull";
+import { BaseScheduler } from "../base-scheduler";
+import {
+  InjectLogger,
+  LoggerService,
+  ProcessSummary,
+} from "@sims/utilities/logger";
+import {
+  getSuccessMessageWithAttentionCheck,
+  logProcessSummaryToJobLogger,
+} from "../../../utilities";
+import { QueueNames } from "@sims/utilities";
+
+@Processor(QueueNames.SIMSToSFASIntegration)
+export class SIMSToSFASIntegrationScheduler extends BaseScheduler<void> {
+  constructor(
+    @InjectQueue(QueueNames.SIMSToSFASIntegration)
+    schedulerQueue: Queue<void>,
+    queueService: QueueService,
+  ) {
+    super(schedulerQueue, queueService);
+  }
+
+  /**
+   * Generate data file consisting of all student and application updates in SIMS since the previous file generation
+   * and send the data file to SFAS.
+   * @param job job.
+   * @returns process summary.
+   */
+  @Process()
+  async generateSFASBridgeFile(job: Job<void>): Promise<string[]> {
+    const processSummary = new ProcessSummary();
+
+    try {
+      this.logger.log(
+        `Processing SIMS to SFAS integration job. Job id: ${job.id} and Job name: ${job.name}.`,
+      );
```

Comment:
I missed it in a previous PR but are we not using as below.
```ts
processSummary.info(
  `Processing application changes report integration job. Job id: ${job.id} and Job name: ${job.name}.`,
);
```

Reply:
Oops! I missed it.